postgres-appliance/launch.sh

```diff
@@ -26,13 +33,15 @@ for i in $(seq 0 7); do
         touch "${PGLOG}/postgresql-$i.csv"
     fi
 done
-chown -R postgres:postgres "$PGROOT" "$RW_DIR/postgresql"
+chown -R postgres: "$PGROOT" "$RW_DIR/certs"
```

Comment:
Because it `$RW_DIR` already has the appropriate permissions based on group membership and `$RW_DIR/postgresql` is created by the run user here https://github.com/ToroNZ/spilo/blob/fd579c00b7895c8b12a8df746a667e7921b9d319/postgres-appliance/launch.sh#L27